hasher.go

```diff
@@ -14,6 +16,11 @@ const (
 
 var _ hash.Hash = (*Hasher)(nil)
 
+var (
+	ErrUnorderedSiblings = errors.New("NMT sibling nodes should be ordered lexicographically by namespace IDs")
+	ErrInvalidNodeLen    = errors.New("Invalid NMT node size")
```

Comment:
[nit][optional] 

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

```suggestion
	ErrInvalidNodeLen    = errors.New("invalid NMT node size")
```

Reply:
Addressed https://github.com/celestiaorg/nmt/pull/113/commits/faf4e5082ded0e955fad0bea46cef7e0c50b89a2